CMakeLists.txt

```diff
@@ -377,8 +382,13 @@ if (BUILD_RDIFF)
   else (POPT_FOUND)
     message (WARNING "Popt library is required for rdiff target")
   endif (POPT_FOUND)
+  target_compile_options(rdiff PRIVATE -Drsync_EXPORTS)
```

Comment:
Are you sure this is needed? As I understand it, when dynamically linking against the librsync.dll you need to NOT define `rsync_EXPORTS` so that it correctly tags the declarations with `__declspec(dllimport)`.

Are you forcing a static build perhaps? If you are, I think this would also be fixed by making librsync_exports.h support static builds.

Reply:
Added `-DLIBRSYNC_STATIC_DEFINE` to `rdiff` target.